cmd/kured/main.go

```diff
@@ -67,6 +69,7 @@ func main() {
 	rootCmd := &cobra.Command{
 		Use:   "kured",
 		Short: "Kubernetes Reboot Daemon",
+		PreRun: flagcheck,
```

Comment:
I am not familiar with cobra, so I trust more experimented folks to determine if PreRun is the right place to evaluate and validate the CLI arguments.

Reply:
I feel it is a good place since it will not be inherited to the children, and the arguments should be loaded also.